Makefile

```diff
@@ -34,7 +34,7 @@ covhtml:
 .PHONY: check
 check: ./golangcilint.yaml
 	golangci-lint --version || curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.26.0
-	golangci-lint --config ./golangcilint.yaml run --enable-all --build-tags extended ./cmd/... ./pkg/...
```

Comment:
containers_image_openpgp? what is this used for?

Reply:
see https://github.com/containers/image#supported-build-tags.

I couldn't build it otherwise.